pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -61,15 +63,25 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 		}
 	}
 
-	// if we are upgradeable==true we can always upgrade
-	up := resourcemerge.FindOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorUpgradeable)
-	if up == nil {
-		klog.V(4).Infof("Precondition %s passed: no Upgradeable condition on ClusterVersion.", pf.Name())
-		return nil
-	}
-	if up.Status != configv1.ConditionFalse {
-		klog.V(4).Infof("Precondition %s passed: Upgradeable %s since %v: %s: %s", pf.Name(), up.Status, up.LastTransitionTime, up.Reason, up.Message)
-		return nil
+	var reason, msg string
+	if deletes := resourcedelete.DeletesInProgress(); len(deletes) > 0 {
```

Comment:
I'd expect this `Run` code to continue to look at the `Upgradeable` condition on the ClusterVersion, and the new `DeletesInProgress` entries to feed that ClusterVersion condition around [here][1].

[1]: https://github.com/openshift/cluster-version-operator/blob/7e7544097149708190903b1f4d088a4529704886/pkg/cvo/upgradeable.go#L221

Reply:
I added a new `Check` into `pkg/cvo/upgradeable.go`.  Please review. 

Assuming that I can back my changes out of `pkg/payload/precondition/clusterversion/upgradeable.go` and no changes are required to that file?